driver/razermouse_driver.h

```diff
@@ -59,8 +61,8 @@
 #define RAZER_MAMBA_TE_ROW_LEN 16       // 0 => 15
 #define RAZER_DIAMONDBACK_ROW_LEN 21    // 0 => 20
 
-#define RAZER_MOUSE_WAIT_MIN_US 600
-#define RAZER_MOUSE_WAIT_MAX_US 800
+#define RAZER_MOUSE_WAIT_MIN_US 31000
+#define RAZER_MOUSE_WAIT_MAX_US 31100
```

Comment:
Can you try to make those conditional only for the devices that need it? I'm not really a fan of massively increasing the time for all devices (and who knows, maybe some other device breaks because of those timeouts, no clue)

Reply:
I tested and submitted a change to keep the lower constants, but let other devices use different values if needed.  Works for me.  Let me know if I need to change anything :)

Reply:
I just noticed you have unique constants declared for other devices in razerkbd_driver.h.  Is that the preferred method?  I could do that too.

Reply:
I suspect that all devices are affected that have the "new" receiver.

Reply:
Added constants that refer to the mouse receiver so they can be used with multiple devices, and the value can be maintained in one spot.

Reply:
Also another thought...maybe use *_NEW_MICE_* instead?  Since the wired device ID counterparts need the longer wait times as well.  Hmmm.